StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -453,6 +453,12 @@ extension PaymentSheet {
                 }
             case .withPaymentMethod(let paymentMethod):
                 confirmWithPaymentMethod(paymentMethod, nil, false)
+            case .withPaymentDetails(let linkAccount, let paymentDetails):
+//                TODO(link): Confirm the last two options should be "nil" and "false"
+                confirmWithPaymentDetails(linkAccount, paymentDetails, nil, false)
+            case .withPaymentMethodParams(let linkAccount, let paymentMethodParams):
+//                TODO(link): Confirm the last two options should be "nil" and "false"
```

Comment:
this comment looks out of date

Reply:
This is intentional, we need to fix this once it's plugged into PaymentSheet. (MOBILESDK-2616)